crates/rolldown/tests/esbuild/default/import_fs_node_common_js/artifacts.snap

```diff
@@ -9,10 +10,10 @@ source: crates/rolldown_testing/src/integration_test.rs
 "use strict";
 
 const fs = __toESM(require("fs"));
-const { default: defaultValue, readFileSync } = __toESM(require("fs"));
+const fs = __toESM(require("fs"));
```

Comment:
this should be deconflicted

Reply:
It cuased by render twice with import star stmt and import named stmt. The pr already two larger, i will fix it at next.

Reply:
https://github.com/rolldown/rolldown/issues/2768